### ISSUE TITLE
Invariant set/manifold finding

### ISSUE THREAD

**Issue by Datseris** [JuliaDynamics/ChaosTools.jl#37]
[Dellnitz, Froyland, Junge - The Algorithms behind `GAIO` - Set Oriented Numerical Methods for Dynamical Systems.pdf](https://github.com/JuliaDynamics/ChaosTools.jl/files/1754193/Dellnitz.Froyland.Junge.-.Unknown.-.The.Algorithms.behind.GAIO.-.Set.Oriented.Numerical.Methods.for.Dynamical.Systems.pdf)

The attached pdf file describes an algorithm (and its implementation) about finding **invariant sets** of a dynamical system.

This is definitely something important and we would definitely want some implementation of that.

They also have an algorithm for "natural/invariant measure" which we could welcome here as well, but I see it as more like "low priority"

**Comment by awage** [comment]
It has been done in Julia already: GAIO.jl

**Comment by Datseris** [comment]
Yes. I wish GAIO.jl would join DynamicalSystems.jl. I've invited them twice throughout the years but declined both times. Perhaps you can reach out to them if you also think there is benefit to having GAIO.jl in DynamicalSystems.jl (probably in Attractors.jl)?

**Comment by Datseris** [comment]
Yes. I wish GAIO.jl would join DynamicalSystems.jl. Perhaps you can reach out to them if you also think there is benefit to having GAIO.jl in DynamicalSystems.jl (probably in Attractors.jl)?

**Comment by awage** [comment]
There are some complementary features: Attractors.jl is good at finding attractors. GAOI.jl is good at describing the attractors. I can always ask but I am not sure they are interested. 

I will have a look at their codebase.

**Comment by Datseris** [comment]
> Attractors.jl is good at finding attractors

Not just attractors anymore. We have two different algorithms for finding the saddle sets.

Not to mention the whole universe of ground breaking functionality for global continuation and nonlocal stability...

From what I can tell they gain much more from joining, so maybe this time they will consider  :D
